benchexec/tablegenerator/react-table/src/components/FilterBox/FilterCard.js

```diff
@@ -23,6 +23,8 @@ const Range = createSliderWithTooltip(Slider.Range);
 
 let debounceHandler = setTimeout(() => {}, 500);
 
+const numericInputDebounce = 800;
```

Comment:
How about using a consistent debounce time? (Above it seems to be set to 500 ms).